cicd/pipelines/creating-applications-with-cicd-pipelines.adoc

```diff
@@ -63,3 +63,5 @@ include::modules/op-triggering-a-pipelinerun.adoc[leveloffset=+1]
 * For more details on pipelines in the *Developer* perspective, see the xref:../../cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc#working-with-pipelines-using-the-developer-perspective[working with Pipelines in the *Developer* perspective] section.
 * To learn more about Security Context Constraints (SCCs), see the xref:../../authentication/managing-security-context-constraints.adoc#managing-pod-security-policies[Managing Security Context Constraints] section.
 * For more examples of reusable tasks, see the link:https://github.com/openshift/pipelines-catalog[OpenShift Catalog] repository. Additionally, you can also see the Tekton Catalog in the Tekton project.
+* For more details on re-encrypt TLS termination, see link:https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html#re-encryption-termination[Re-encryption Termination].
+* For more details on secured routes, see the xref:../../networking/routes/secured-routes.adoc#secured-routes[Secured routes] section.
```

Comment:
`#secured-routes` in URL is unnecessary

Reply:
@ppitonak Thanks for the comment. Please note that this is the convention we follow across the documentation when giving reference links - whether it is a link to the page or to the specific section. So retaining as is. Thanks!